cloudify-manager-worker/templates/before_hook.yaml

```diff
@@ -23,6 +24,11 @@ data:
       rm -f /etc/cloudify/config.yaml
       cp -a --verbose "$CLOUDIFY_DATA_DIR/etc/config.yaml" /etc/cloudify/config.yaml
 
+      if [ -f "$OKTA" ]; then
+        mkdir -p /etc/cloudify/ssl
+        cp -a --verbose "$OKTA" /etc/cloudify/ssl/okta_certificate.pem
```

Comment:
sure we need it?

Reply:
I mean it's static cert, will not be changed, why you want to save/copy it to persistent volume?

Reply:
Because cloudify has the SAML certPath hardcoded to okta_certificate.pem, so it's either copying it or linking it.